Lua/Lua.sublime-syntax

```diff
@@ -423,8 +423,8 @@ contexts:
           |next|pairs|pcall|print|rawequal|rawget|rawlen|rawset|select
           |setmetatable|tonumber|tostring|type|xpcall
           |require|getfenv|module|setfenv|unpack
-        ){{identifier_break}}
-      scope: support.function.builtin.lua
+        ){{function_call_ahead}}
```

Comment:
I actually think this match might need to be doubled, because if there is a function call ahead, we apply `meta.function-call`, but if there isn't, the function is just as an object where it should *still* be scoped as `support.function`. Non-deterministic highlighting *might* also help.

Reply:
I actually think is odd to highlight any variable with these names, because as a Lua programmer is common for me to declare a local variable named `next`, `type`, `module` in certain circumstances and they are certainly not the builtin functions on these cases so are wrongly highlighted. Thus was intentional to change this.

Plus the highlighter already can't (and shouldn't) highlight non builtin function references without a call ahead, so doing the same for builtin functions keeps the highlighting more uniform.